Core/src/org/sleuthkit/autopsy/report/infrastructure/ReportVisualPanel1.java

```diff
@@ -216,6 +233,27 @@ PortableCaseReportModule getPortableCaseModule() {
         return null;
     }
 
+    /**
+     * Get updated configuration for all report modules.
+     *
+     * @return
+     */
+    Map<String, ReportModuleConfig> getUpdatedModuleConfigs() {
```

Comment:
Can't this be private?

Reply:
@rcordovano per our discussion, we will leave code as is.